CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
-## TBD
-* Updated `hotels-oss-parent` version to 4.2.0 (was 4.0.0). Contains updates to the copyright header.
+## [15.0.0] - TBD
+
+### Fixed
+* Table properties can now be added to default transformations.
+
+### Changed
+* `hotels-oss-parent` version to 4.2.0 (was 4.0.0). Contains updates to the copyright header.
+* Default `avro-serde-options` must now be included within `transform-options`.
```

Comment:
This is the biggest change impacting downstream users so put the Changed block first and then this line first within the block.

Reply:
Add a note that this is backwards incompatible config change.

Reply:
Link to avro README file for the example.

Reply:
Done